sechub-api-java/src/main/java/com/mercedesbenz/sechub/api/java/demo/cli/SecHubClientConfigurationFactory.java

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.api.java.demo.cli;
+
+import java.net.URI;
+
+import com.mercedesbenz.sechub.api.java.SecHubClient;
+import com.mercedesbenz.sechub.api.java.demo.cli.exceptions.SecHubClientConfigurationRuntimeException;
+
+public class SecHubClientConfigurationFactory {
+
+    EnvironmentVariableReader environmentVariableReader;
+
+    public SecHubClientConfigurationFactory() {
+        environmentVariableReader = new EnvironmentVariableReader();
+    }
+
+    public SecHubClient create(CommandLineSettings settings) {
+        URI serverUri = null;
+        if (settings.getServerUri() != null)
+            serverUri = URI.create(settings.getServerUri());
+
+        int serverPort = settings.getServerPort();
+        String userId = settings.getUserId();
+        String apiToken = settings.getApiToken();
+
+        String envServerUri = environmentVariableReader.readAsString(EnvironmentVariableConstants.SECHUB_SERVER_URI);
```

Comment:
Instead of having a separate `SECHUB_SERVER_URI` and `SECHUB_SERVER_PORT` variables use a combined `SECHUB_SERVER` variable. It is easier for the user. The same applies to the command line.

The `sechub-api.sh` script expects a single `SECHUB_SERVER` argument. It would be nice if your demo app would work the same way.